dockers/docker-snmp-sv2/snmpd

```diff
@@ -8,4 +8,4 @@ export MIBS=
 SNMPDRUN=yes
 
 # snmpd options (use syslog, close stdin/out/err).
-SNMPDOPTS='-LS4d -Lf /dev/null -u snmp -g snmp -I -smux,mteTrigger,mteTriggerConf,ifTable,ifXTable -p /run/snmpd.pid'
+SNMPDOPTS='-LS4d -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux,mteTrigger,mteTriggerConf,ifTable,ifXTable -p /run/snmpd.pid'
```

Comment:
I thought so. But in the snmp docker /etc/passwd and owner of /var/lib/snmp shows Debian-snmp.

---
In reply to: [99451792](https://github.com/Azure/sonic-buildimage/pull/252#discussion_r99451792) [](ancestors = 99451792)